egs2/TEMPLATE/tts1/tts.sh

```diff
@@ -333,68 +339,81 @@ if ! "${skip_data_prep}"; then
                 "data/${dset}/wav.scp" "${data_feats}${_suf}/${dset}"
             echo "${feats_type}" > "${data_feats}${_suf}/${dset}/feats_type"
         done
+    fi
 
-        # Extract X-vector
+    # Extract X-vector
+    if [ ${stage} -le 3 ] && [ ${stop_stage} -ge 3 ]; then
```

Comment:
I understand the motivation of separating this stage.
However, we need to fix all of the documents, which is too heavy.
Could you keep it as it?